### PR TITLE
Add card holder name.

### PR DESCRIPTION
**Pull request by yatatsu**
In Token API, we can pass card holder name as `card[name]` but payjp-ios doesn' t.

cf. https://pay.jp/docs/api/#%E3%83%88%E3%83%BC%E3%82%AF%E3%83%B3%E3%82%92%E4%BD%9C%E6%88%90 (API Reference)

This PR added argument for card holder name as following.

```Example.swift
let apiClient = APIClient(publicKey: "pk_test_d5b6d618c26b898d5ed4253c")
apiClient.createToken(with: "4242424242424242",
                      cvc: "123",
                      expirationMonth: "02",
                      expirationYear: "2020",
                      name: "TARO YAMADA")
```

~~This change **WILL NOT** break compatibility, because `name` is `nil` by default.~~
The default argument is not efficient if you use Objective-C.

### CODE REVIEW

File: example-swift/example-swift/ViewController.swift [hunk 1]
```diff
@@ -53,12 +54,14 @@ class ViewController: UITableViewController {
         let cvc = fieldCardCvc.text ?? ""
         let month = fieldCardMonth.text ?? ""
         let year = fieldCardYear.text ?? ""
-        print("input number=\(number), cvc=\(cvc), month=\(month), year=\(year)")
+        let name = filedCardName.text
```
Comment (vc7) [inline example-swift/example-swift/ViewController.swift]: Please provide a default value for name to prevent warning coming out from the print statement.

**Review by vc7 — COMMENTED** [review]
Thanks for the effort! 
Please take a look at the comment :)

**Comment by yatatsu** [comment]
@vc7 Thanks! I fixed sample code warning, and I also fix example-swift project warnings. 
Could you review it again?

**Review by vc7 — APPROVED** [review]
LGTM!

**Pull request merged**